src/components/Story/StoryFull.js

```diff
@@ -144,6 +156,49 @@ class StoryFull extends React.Component {
       );
     }
 
+    const popoverMenu = [
+      <PopoverMenuItem key="follow" disabled={pendingFollow}>
+        {pendingFollow ? <Icon type="loading" /> : <i className="iconfont icon-people" />}
+        {followText}
+      </PopoverMenuItem>,
+      <PopoverMenuItem key="save">
+        {pendingBookmark ? <Icon type="loading" /> : <i className="iconfont icon-collection" />}
+        <FormattedMessage
+          id={postState.isSaved ? 'unsave_post' : 'save_post'}
+          defaultMessage={postState.isSaved ? 'Unsave post' : 'Save post'}
+        />
+      </PopoverMenuItem>,
+      <PopoverMenuItem key="report">
+        <i className="iconfont icon-flag" />
+        <FormattedMessage id="report_post" defaultMessage="Report post" />
+      </PopoverMenuItem>,
+    ];
+
+    if (ownPost) {
+      const postMetaData = jsonParse(post.json_metadata);
+      let item = null;
+
+      if (postMetaData.format !== 'markdown') {
```

Comment:
Why limiting the edit to markdown?

Reply:
This was handled this way in Busy1.
I will change this.